2020_Fall/readme.md

```diff
@@ -71,7 +71,7 @@ Make the Pull Request from your own GitHub account. Make sure you're changing th
 
 - [ ] Kimball, Ellen
 
-- [ ] Liew, Jeffrey
+- [x] Liew, Jeffrey
```

Comment:
@baldjeff Looks like you've missed adding the GitHub username and emoji. Just make another commit on this branch.